python/tidy/servo_tidy/tidy.py

```diff
@@ -48,6 +48,49 @@
 # File patterns that are ignored for all tidy and lint checks.
 FILE_PATTERNS_TO_IGNORE = ["*.#*", "*.pyc"]
 
+# Files that are ignored for all tidy and lint checks.
+IGNORED_FILES = [
```

Comment:
This variable is no longer referenced anywhere in `tidy.py`, meaning this list is not used to ignore files.